TTS/trainer.py

```diff
@@ -622,6 +632,7 @@ def _optimize(
         grad_norm = 0
         update_lr_scheduler = True
         if self.use_amp_scaler:
+            # TODO Capacitron mixed precision
```

Comment:
In my earlier experiments, mixed precision somehow broke Capacitron, but I was also not focusing on that issue greatly. I'd appreciate some help and input on this one. :)

Reply:
in what way it breaks the training more specifically?

Reply:
I forgot what exactly the problem was but it just didn't work (no alignment and non-intelligable speech.

Reply:
ok then let's keep it for after the first merge.